### PR TITLE
Add Locations API

### PR DESCRIPTION
**Pull request by baisa**
What
Get the new Locations API application running in govuk-docker.

Why
This will allow us to develop the application locally and test the use of it's API (when implemented) in other GOV.UK applications.

[Trello card](https://trello.com/c/KZrHQ6gT/2691-get-locations-api-working-in-govuk-docker-3)

### CODE REVIEW

**Review by brucebolt — APPROVED** [review]
This runs perfectly on my machine, great work @baisa.

Could I just make a small nitpick request before you merge: could the two commits be squashed into one, as the second commit solely makes a correction to the first so it is a little redundant.

**Comment by baisa** [comment]
Good point @brucebolt, I have squashed the commits.

**Comment by brucebolt** [comment]
> Good point @brucebolt, I have squashed the commits.

Great, thank you.  Feel free to merge this PR when you are ready.

**Pull request merged**
